python/codepipeline-docker-build/requirements.txt

```diff
@@ -0,0 +1,49 @@
+attrs==19.2.0
+aws-cdk.assets==1.12.0
```

Comment:
We don't hard-code the version of the CDK libraries  - just leave it as `aws-cdk.assets`. Same for the other dependencies.